.env.production-goerli

```diff
@@ -0,0 +1,65 @@
+## all
+NODE_ENV = 'production'
+LOG_DESTINATION = 'console'
+LOG_LEVEL = 'info'
+# e2e-tests
+BROWSER_LOG_DESTINATION = 'browser.log'
+HEADLESS = 'true'
+USE_DAPPETEER = 'false'
+WEB3TORRENT_URL = 'https://web3torrent.statechannels.org'
+CLOSE_BROWSERS = 'true'
+
+## web3torrent, xstate-wallet, rps, simple-hub
+CHAIN_NETWORK_ID = '5'
+
+## simple-hub, e2e
+INFURA_API_KEY = '5cc24072eb7141ada6dd9d3cabf4224d'
+RPC_ENDPOINT = https://goerli.infura.io/v3/${INFURA_API_KEY}
+
+## simple-hub, rps
+FIREBASE_API_KEY = 'AIzaSyAOvhDzJir_El3O6SJ2xQlrpOisnObq6zw'
+FIREBASE_PREFIX = 'netlify-goerli'
+FIREBASE_PROJECT = 'rock-paper-scissors-production'
+
+## rps, xstate-wallet, ttt
+TARGET_NETWORK = 'goerli'
+
+## web3torrent, rps, ttt
+WALLET_URL = 'https://xstate-wallet.statechannels.org'
+
+## simple-hub, xstate-wallet
+CONSENSUS_APP_ADDRESS = '0xdb7accd228222ea0035cb3da82f09eb6fde4f7ae'
+
+### NitroAdjudicator and ETHAssetHolder deployed by George Knee, by uploading files to https://remix.ethereum.org/ and using compiler 0.6.10+commit.00c0fcaf, optimization ON. Source code verified on https://goerli.etherscan.io/ .
+NITRO_ADJUDICATOR_ADDRESS = '0xedae380e8da298979848337d7527f441071e68cf'
+ETH_ASSET_HOLDER_ADDRESS = '0x8e409FB551fC96f5129cb41e7194e8F8CB9df0a0'
+
+TRIVIAL_APP_ADDRESS = '0x93ae13bd37021cc4e17685578851bcf152ee28d5'
+
+## simple-hub
+FIREBASE_URL = https://${FIREBASE_PROJECT}.firebaseio.com/
+HUB_CHAIN_PK = '<override this on private heroku dashboard>'
+HUB_PARTICIPANT_ID = 'firebase:simple-hub'
+
+## xstate-wallet
+# hub address is from Heroku which specifies a funded account on Goerli (kept secret here)
+HUB_DESTINATION = '0x0000000000000000000000004F72D4213411Faa1b1b7bf43f9F36667A98B973A'
+USE_INDEXED_DB = 'true'
+
+## web3torrent
+FUNDING_STRATEGY = 'Virtual'
+SINGLE_ASSET_PAYMENT_CONTRACT_ADDRESS = '0x5cb071f46fb06333f33dcae7aaf8c2606d7a078d'
+SKIP_PREFLIGHT_CHECK = 'true'
+TRACKER_URL = 'web3torrent-tracker.statechannels.org'
+TRACKER_URL_HTTP_PROTOCOL = 'https'
+GENERATE_SOURCEMAP = 'true'
+
+## rps
+RPS_CONTRACT_ADDRESS = '0x'
```

Comment:
We are there configuration options for packages in the `apps` repo?

Reply:
Oversight... will remove

Reply:
0642471